js/src/DappRequests/store.js

```diff
@@ -276,12 +268,20 @@ export default class Store {
       this.executePubsubCall(data, source);
     } else if (subId) {
       subId === '*'
-        ? this.provider.unsubscribeAll().then(v => this._methodCallbackPost(id, from, source, token)(null, v))
-        : this.provider.unsubscribe(subId).then(v => this._methodCallbackPost(id, from, source, token)(null, v));
+        ? this.provider
+          .unsubscribeAll()
+          .then(v =>
+            this._methodCallbackPost(id, from, source, token)(null, v)
```

Comment:
Could be extracted outside of the ternary.

Reply:
Fixed